push-build.sh

```diff
@@ -19,37 +19,42 @@ PROG=${0##*/}
 ########################################################################
 #+
 #+ NAME
-#+     $PROG - Push Continuous Integration Build
+#+     $PROG - Push Kubernetes Release Artifacts up to GCS
 #+
 #+ SYNOPSIS
-#+     $PROG  [--nomock] [--federation] [--noupdatelatest]
+#+     $PROG  [--nomock] [--federation] [--noupdatelatest] [--ci]
 #+            [--bucket=<alt GS bucket>] [--bucket-mirror=<mirror GS bucket>]
 #+     $PROG  [--helpshort|--usage|-?]
 #+     $PROG  [--help|-man]
 #+
 #+ DESCRIPTION
-#+     Replaces kubernetes/build/push-ci-build.sh.
-#+     Used for pushing CI builds during Jenkins' runs.
-#+     Runs in mock mode by default. Use --nomock to do a real push.
+#+     Replaces kubernetes/build/push-*-build.sh.
+#+     Used for pushing developer builds and Jenkins' continuous builds.
+#+
+#+     Developer pushes simply run as they do pushing to devel/ on GCS.
+#+     In --ci mode, $PROG runs in mock mode by default.  Use --nomock to do
+#+     a real push.
 #+
 #+     Federation values are just passed through as exported global vars still
 #+     due to the fact that we're still leveraging the existing federation 
 #+     interface in kubernetes proper.
 #+
 #+ OPTIONS
-#+     [--nomock]                - Enables a real push
+#+     [--nomock]                - Enables a real push (--ci only)
```

Comment:
mock mode just adds a $USER namedspace to the GCS bucket.  Since the nature of a 'devel' push is unique anyway, there's no point in offereing a 'mock' solution for '-devel' so its just being removed.  Leaving it in for -ci- usage, though probably not to be used regularly.